src/Tizen.NUI/src/public/RenderEffects/BackgroundBlurEffect.cs

```diff
@@ -0,0 +1,96 @@
+
+
+using System.ComponentModel;
+using System;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// Background blur effect of a View.
+    /// Applications can apply BackgroundBlurEffect as the example below :
+    /// <code>
+    /// BackgroundBlurEffect effect = new BackgroundBlurEffect();
+    /// view.SetRenderEffect(effect);
+    /// </code>
+    /// </summary>
+public class BackgroundBlurEffect : RenderEffect
+{
+    internal BackgroundBlurEffect(global::System.IntPtr cPtr) : base(cPtr)
+    {
+    }
+
+    /// <summary>
+    /// The property determines whether to render the effect only once or at every frame update.
+    /// </summary>
+    /// <since_tizen> 13 </since_tizen>
+    public bool BlurOnce
+    {
+        get
+        {
+            bool blurOnce = Interop.BackgroundBlurEffect.GetBlurOnce(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return blurOnce;
+        }
+
+        set
+        {
+            Interop.BackgroundBlurEffect.SetBlurOnce(SwigCPtr, value);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+    }
+
+    /// <summary>
+    /// The property is blur radius value.
+    /// The unit is pixel, but the property is in float type since many other platforms use float for blur effect radius.
+    /// </summary>
+    /// <since_tizen> 13 </since_tizen>
```

Comment:
```suggestion
    /// <since_tizen> 13 </since_tizen>
    [EditorBrowsable(EditorBrowsableState.Never)]
```

Add browsable never, since ACR process is slow.